frontend/www/js/omegaup/components/arena/RunDetails.vue

```diff
@@ -336,24 +363,33 @@ export default class ArenaRunDetails extends Vue {
     caseName: string,
     caseType: string,
   ): string {
-    return cases[caseName] ? cases[caseName][caseType] : '-';
+    switch (caseType) {
+      case 'in':
+        return cases[caseName]?.in ?? '-';
+      case 'out':
+        return cases[caseName]?.out ?? '-';
+      case 'contestantOutput':
+        return cases[caseName]?.contestantOutput ?? '-';
+    }
+    return '-';
```

Comment:
```suggestion
     return cases[caseName]?.[caseType].in ?? EMPTY_FIELD;
```

el operador `?.` también funciona con arreglos! https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

y algo que se me olvidó pedir antes: dado que `-` es un caracter completamente válido en una entrada o salida, se puede en vez declarar una constante en L343:

```typescript
const EMPTY_FIELD = '∅';
```

? opcionalmente se puede hacer que si se muestra este símbolo, se agregue una clase `empty` para desplegar el símbolo en fuente normal, un poco más grande y centrado vertical+horizontalmente en la celda? así es súper claro que algo estuvo mal.

Reply:
Listo. Sólo que me marca el siguiente error:
```
381:12 Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ contestantOutput?: string | undefined; in: string; out: string; }'.
      No index signature with a parameter of type 'string' was found on type '{ contestantOutput?: string | undefined; in: string; out: string; }'.
        379 |     caseType: string,
        380 |   ): string {
      > 381 |     return cases[caseName]?.[caseType] ?? EMPTY_FIELD;
            |            ^
        382 |   }
        383 |
        384 |   showDiff(cases: types.ProblemCasesContents, caseName: string): string {
```